kubernetes/samples/scripts/create-rcu-schema/common/template/rcu.yaml.template

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# This is a template for RCU Pod
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: rcu
+  name: rcu
+  namespace: %NAMESPACE%
+spec:
+  containers:
+  - args:
+    - sleep
+    - infinity
+    image: container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+    imagePullPolicy: IfNotPresent
```

Comment:
In our k8s multi nodes env we use "imagePullPolicy: Always" to pull the latest FMW image. So if "imagePullPolicy" is customizable, it would be nicer.

Reply:
Hi @maggiehe00 ,

Updated scripts to make "imagePullPolicy" customizable.
Please review.

Thanks,
Asha